### DISCUSSION TITLE
Unneeded .cue sheets cause trouble

### DISCUSSION THREAD

**Discussion by aljus7** [rern/rAudio#591] [Bug Reports]
Is there a possible fix, so those *.flac files would be playable without removing *.cue files.
Example: https://mega.nz/folder/H10zXIwQ#tuvRs6j6eZmZGrvx6xB9-w

**Comment by rern** [comment]
This is not a bug. rAudio plays albums with `*.cue` fine. It even saves individual tracks to playlists properly.

Your album:
- Your folder has a single `*.cue` with multiple `*.flac` which is not normal.
- Tracks in the `*.cue` are not listed with successive `INDEX`
- This `*.cue`  is not useful for anything but trouble.

**Comment by fsttrck** [comment]
I have similar issues with flac files playback when there are .cue files in same folder as individual flac files.
Would it be too much to ask for the option to ignore .cue on library scan be available please?
Thank you

**Comment by fsttrck** [comment]
Something like below 
![image](https://github.com/user-attachments/assets/766bac3b-8e76-45c6-85fb-9c9d0bd7262a)

**Reply by rern** [reply]
Please explain why you still need the `*.cue` file when each file has nothing to do with it.

**Comment by fsttrck** [comment]
Mainly to burn to CDs, however it's easier to store it on the NAS as individual tracks NAS then can serve it to whatever player.

The .cue file is basically the table of contents extracted from a CD when it was ripped. It contains information on the CD structure, the tracks, what mode they are, as well as CD Text etc.

With the CUE file you can recreate the exact structure of the original CD. Including all the CD-TEXT, serial numbers and any data tracks that may be on a mixed mode CD (mixed mode CD's contain audio and data tracks). It would also recreate the pregaps between tracks and (if you have the audio) any hidden Easter egg audio or hidden tracks.

**Reply by rern** [reply]
Try:
- Create a file `.mpdignore` in each folder.
- Put `FILENAME.cue` in the file. (Replace `FILENAME` with the actucal name)
- Update Library
